text/1071-cell.md

```diff
@@ -0,0 +1,296 @@
+project-link:
+suite: 
+---
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+<-- Replace "RFC title" with the title of your RFC -->
+# Introduce `Cell` 
+
+## Summary
+
+This RFC introduces a new tracking primitive, which represents a single value, the `Cell`.
+
+## Motivation
+
+The `Cell` is part of the "spreadsheet analogy" when talking about reactivity -- it represents a single tracked value, and can be created without the use of a class, making it a primate candidate for demos[^demos] and for creating reactive values in function-based APIs, such as _helpers_, _modifiers_, or _resources_. They also provide a benefit in testing as well, since tests tend to want to work with some state, the `Cell` is wholly encapsulated, and can be quickly created with 0 ceremony. 
+
+This is not too dissimilar to the [Tracked Storage Primitive in RFC#669](https://github.com/emberjs/rfcs/blob/master/text/0669-tracked-storage-primitive.md). The `Cell` provides more ergonomic benefits as it doesn't require 3 imports to use. 
+
+The `Cell` was prototyped in [Starbeam](https://starbeamjs.com/guides/fundamentals/cells.html) and has been available for folks to try out in ember via [ember-resources](https://github.com/NullVoxPopuli/ember-resources/tree/main/docs/docs). 
+
+[^demos]: demos _must_ over simplify to bring attention to a specific concept. Too much syntax getting in the way easily distracts from what is trying to be demoed. This has benefits for actual app development as well though, as we're, by focusing on concise demo-ability, gradually removing the amount of typing needed to create features. 
+
+## Detailed design
+
+
+### Types 
+
+Some interfaces to share with future low-level reactive primitives:
+
+```ts
+
+interface Reactive<Value> {
+    /**
+    * The underlying value
+    */
+    current: Value;
+    /**
+    * Returns the underlying value
+    */
+    read(): Value;
+}
+
+interface ReadOnlyReactive<Value> extends Reactive<Value> {
+    /**
+    * The underlying value.
+    * Cannot be set.
+    */
+    readonly current: Value;
+
+    /**
+    * Returns the underlying value
+    */
+    read(): Value;
+}
+
+interface Cell<Value> extends Reactive<Value> {
+    /**
+    * Utility to create a Cell without the caller using the `new` keyword.
+    */
+    static create<T>(initialValue: T, equals?: (a: T, b: T) => boolean): Cell<T>;
+
+    /**
+    * The constructor takes an optional initial value and optional equals function.
+    */
+    constructor(initialValue?: Value, equals?: (a: Value, b: Value) => boolean): {}
+
+    /**
+    * Function short-hand of updating the current value
```

Comment:
"short-hand" suggests to me there is also another way to set, but there is no, right? (other than `update`, but I would agree that is a "short-hand" for `set`)

Reply:
assigning a new value to `current` is the other way

Reply:
(I added clarity here)